cmd/ocsp-updater/main.go

```diff
@@ -228,12 +237,14 @@ func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Ti
 			 FROM certificateStatus AS cs
 			 JOIN certificates AS cert
 			 ON cs.serial = cert.serial
-			 WHERE cs.ocspLastUpdated < :lastUpdate
+			 WHERE cs.ocspLastUpdated > :maxAge
```

Comment:
Since `sa.AddCertificate` still sets the `ocspLastUpdated` field to `0` won't we get the same issue as you encountered with the test with every new certificate we add to the database? (i.e. a initial OCSP response will never be generated)

Perhaps this should be `WHERE (cs.ocspLastUpdated > :maxAge OR cs.ocspLastUpdated = 0)`?